cantabular/queries.go

```diff
@@ -268,31 +268,32 @@ func (data *QueryData) Encode(query string) (bytes.Buffer, error) {
 func (c *Client) queryUnmarshal(ctx context.Context, graphQLQuery string, data QueryData, v interface{}) error {
 	url := fmt.Sprintf("%s/graphql", c.extApiHost)
 
+	logData := log.Data{
+		"url":        url,
+		"query":      graphQLQuery,
+		"query_data": data,
+	}
+
 	res, err := c.postQuery(ctx, graphQLQuery, data)
-	defer closeResponseBody(ctx, res)
 	if err != nil {
 		return err
```

Comment:
You need to expand this error to include the logData etc.. Pretty sure you did this before?

Reply:
I have not redefined any errors, it was like that before. I'll expand however, one second

Reply:
Done